frontstage/views/secure_messaging.py

```diff
@@ -69,15 +69,17 @@ def create_message(session):
         if request.form['submit'] == 'Save draft':
             if "msg_id" in request.form:
                 data['msg_id'] = request.form['msg_id']
+                headers['Authorization'] = request.cookies['authorization']
                 response = requests.put(DRAFT_PUT_API_URL.format(request.form['msg_id']), data=json.dumps(data), headers=headers)
```

Comment:
for some reason I've been able to hit this line with an empty `msg_id` which send you to endpoint `/draft//modify` which returns a 404